osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -214,6 +224,8 @@ private double computeAccuracyValue(ScoreInfo score, OsuDifficultyAttributes att
                 accuracyValue *= 1.14;
             else if (score.Mods.Any(m => m is OsuModHidden))
                 accuracyValue *= 1.08;
+            else if (score.Mods.Any(m => m is OsuModTraceable))
+                accuracyValue *= 1.08;
```

Comment:
```suggestion
            else if (score.Mods.Any(m => m is OsuModHidden || m is OsuModTraceable))
                accuracyValue *= 1.08;
```

apply this to others to avoid duplication